src/ebb.ts

```diff
@@ -108,6 +108,11 @@ export class EBB {
     }
   }
 
+  /** Send the new General Query command to get limit switch, button and fifo status */
+  public async queryGeneral(): Promise<number> {
+    return parseInt(await this.query("QG"), 16);
```

Comment:
If I'm understanding correctly, this is only available in the [`3.0` firmware](https://evil-mad.github.io/EggBot/ebb.html#version_differences), right?

If so, this should probably check the firmware version first: https://github.com/nornagon/saxi/blob/31f59ac8cf087e4c157b65be398fd210fa81615b/src/ebb.ts#L23